firebase-database/src/main/java/com/google/firebase/database/core/utilities/DefaultRunLoop.java

```diff
@@ -128,13 +128,19 @@ public static String messageForException(Throwable t) {
           + "https://firebase.google.com/docs/database/ios/structure-data#best_practices_for_data_structure"
           + " and "
           + "https://firebase.google.com/docs/database/android/retrieve-data#filtering_data";
+    } else if (t instanceof NoClassDefFoundError) {
+      return "A symbol that the Firebase Database SDK depends on failed to load. This usually "
+          + "indicates that your project includes an incompatible version of another Firebase "
+          + "dependency. If updating your dependencies to the latest version does not resolve "
+          + "this issue, please file a report at https://github.com/firebase/firebase-android-sdk";
     } else if (t instanceof DatabaseException) {
       // Exception should be self-explanatory and they shouldn't contact support.
       return "";
     } else {
       return "Uncaught exception in Firebase Database runloop ("
           + FirebaseDatabase.getSdkVersion()
-          + "). Please report to firebase-database-client@google.com";
+          + "). If this is not a problem with your app, please file an issue at "
+          + " https://github.com/firebase/firebase-android-sdk";
```

Comment:
FWIW- You could add something about "make sure you're using the latest version of the SDK" if you want to potentially decrease noise level a little more.

Reply:
Added